apps/calendar/js/views/view_event.js

```diff
@@ -83,10 +82,8 @@ ViewEvent.prototype = {
     this.setContent('location', model.location);
 
     if (this.originalCalendar) {
-      // Set calendar color.
-      this.element
-        .querySelector('section[data-type="list"]')
-        .className = 'calendar-id-' + model.calendarId;
+      this.element.querySelector('.icon-calendar-dot').style.color =
```

Comment:
I think `querySelector` is a better option since it returns a single element and it's clear enough.